docs/terms/metric_store.md

```diff
@@ -3,12 +3,51 @@ id: metric_store
 title: Metric Store
 hoverText: A connector to store and retrieve information about computed attributes of data, such as the mean of a column.
 ---
+
 import UniversalMap from '/docs/images/universal_map/_universal_map.mdx';
+import TechnicalTag from '../term_tags/_tag.mdx';
+import SetupHeader from '/docs/images/universal_map/_um_setup_header.mdx'
+import CreateHeader from '/docs/images/universal_map/_um_create_header.mdx';
+import ValidateHeader from '/docs/images/universal_map/_um_validate_header.mdx';
+
+
+<UniversalMap setup='active' connect='active' create='active' validate='active'/> 
 
-<UniversalMap setup='active' connect='active' create='active' validate='active'/>
+## Overview
+
+### Definition
 
 A Metric Store is a connector to store and retrieve information about computed attributes of data, such as the mean of a column.
 
+### Features and promises
+
+The Metric Store differs from an Evaluation Parameter Store in how it formats its data.  Information stored in a Metric Store is kept in a format that more easily converted into tables which can be used for reports or to analyze trends.  It can also be referenced as values for Evaluation Parameters.  
+
+### Relationship to other objects
+
+A Metric Store can be referenced by an Expectation Suite to populate values for Evaluation Parameters used by Expectations within that suite.  Metric Stores are also used in Checkpoints to store Metrics that are included in the Validation Results which are passed to the Checkpoints' `action_list`.
+
+## Use cases
+
+<SetupHeader/>
+
+If you intend to use a Metric Store, you can configure it in your `great_expectations.yml` file when you configure other Stores.  A Metric Store is an optional addition to `great_expectations.yml`, and one will not be included by default when you first initialize your Data Context.
```

Comment:
It might be useful to either include an example config or link to another page that has more details on setting up a metric store

Reply:
Link added.